include/rive/span.hpp

```diff
@@ -61,6 +62,12 @@ template <typename T> class Span {
     }
 };
 
+template <typename Container>
+inline auto toSpan(Container& c)
+        -> Span<typename std::remove_reference<decltype(*(c.data()))>::type> {
```

Comment:
Hah, nice gymnastics. This is all just to extract Container's templated type right? e.g. nab int from vector<int>